src/lib/api/org/index.ts

```diff
@@ -179,9 +196,72 @@ export async function listProjects(
           JSON.stringify({ data: res.data, status: statusCode }),
       );
     }
-    return res.data || {};
+    
+    let v3responseData = res.data as v3ProjectsResponse
+    let projectData = convertToSnykProject(orgId.trim(), v3responseData)
+
+    while (v3responseData.links.next) {
+      const res = await requestManager.request({
+        verb: 'get',
+        url: v3responseData.links.next,
+        body: JSON.stringify(filters),
+        useRESTApi: true,
+      });
+  
+      const statusCode = res.statusCode || res.status;
+      if (!statusCode || statusCode !== 200) {
+        throw new Error(
+          'Expected a 200 response, instead received: ' +
+            JSON.stringify({ data: res.data, status: statusCode }),
+        );
+      }
+      v3responseData = res.data as v3ProjectsResponse
+      const projectDataTmp = projectData
+      projectData = convertToSnykProject(orgId.trim(), res.data, projectDataTmp)
+    }
+
+    return projectData || {};
   } catch (e) {
     debug('Failed to update notification settings for ', orgId, e);
     throw e;
   }
 }
+
+export function convertToSnykProject(orgId: string, v3Response: v3ProjectsResponse, v3PreviousDataConverted?: ProjectsResponse): ProjectsResponse {
+
+  let snykProjectData = {} as ProjectsResponse 
+  
+  snykProjectData = {
+    org: {
+      name: "",
```

Comment:
why is this empty? This seems like invalid data

Reply:
it's empty because we don't have this info in the v3 api

Reply:
is it used anywhere? Try removing it and see?